tsconfig.json

```
@@ -19,6 +19,7 @@
       "bazel-bin",
       "bazel-out/k8-opt/bin",
       "bazel-out/k8-fastbuild/bin",
+      "bazel-out/platform_linux-fastbuild/bin",
```

can you double-check that this works with `-c opt`? you might need to add this:

```suggestion
      "bazel-out/platform_linux-opt/bin",
      "bazel-out/platform_linux-fastbuild/bin",
```

---

Good point. I pushed a fix.